### ISSUE TITLE
How should one install/setup repack in windows?

### ISSUE THREAD

**Issue by arteika** [reorg/pg_repack#243]
I managed to get pg_repack compiled under windows with these files being generated by the visual studio:

```
pg_repack\msvc\bin\x64\8.3\bin\pg_repack.exe
pg_repack\msvc\bin\x64\8.3\lib\pg_repack.dll
pg_repack\msvc\bin\x64\8.3\lib\pg_repack.exp
pg_repack\msvc\bin\x64\8.3\lib\pg_repack.lib
```

I placed them into the postgres (9.6) bin 

Now I need to generate pg_repack.control file for windows. I found the directory `pg_repack\lib` with 

```
pg_repack\lib\pg_repack.sql.in
pg_repack\lib\pg_repack.control.in
```

so I suppose I need to replace `REPACK_VERSION` and `RELHASOIDS`? Do you have maybe an example?

**Comment by dvarrazzo** [comment]
This package is not designed to be built and run on windows. At the moment it doesn't look like the developers have plenty of bandwidth to maintain it as it is: I doubt they will want to maintain the window version too.

**Comment by arteika** [comment]
Hi @dvarrazzo ,

Thanks for the infos. I understand that it takes quite a bit of effort to maintain the package. Let me see how further I can get with my own initiative and how much of support and in what form we can get for this project.

As of now I managed to get to this point:

- I took the pg_repack.sql, replaced the MODULE_PATHNAME with the path of the DLL, and created the corresponding mappings between the SQL and C Functions like this:

```
CREATE FUNCTION repack.version() RETURNS text AS
'"C:\Program Files\PostgreSQL\9.6\lib\pg_repack', 'repack_version'
LANGUAGE C IMMUTABLE STRICT;
```

- I loaded the DLL using the 

```
load 'C:\Program Files\PostgreSQL\9.6\lib\pg_repack.dll'
```

- Simple test of the SQL Functions works (executing version function gives me `'unknown'` as the answer)

Executing pg_repack.exe from a command line does not produce any output, independent of an existing or misspelled table:

```
pg_repack.exe -h localhost -d play -p 5432 --table play.rxx
pg_repack.exe -h localhost -d play -p 5432 --table play.r
```

Debugging project in visual studio 2017 gives a reference error (in stack I see that pg_repack launches a couple of functions from the postgres.exe).

Does that mean that the target of pg_repack needs to match the target of the postgres?
